doctr/transforms/modules/base.py

```diff
@@ -85,3 +86,22 @@ def __call__(self, img: Any) -> Any:
         if random.random() < self.p:
             return self.transform(img)
         return img
+
+
+class RandomRotate(NestedObject):
+    """Randomly rotate a tensor image
+
+    Args:
+        max_angle: maximum angle for rotation, in degrees. Angles will be uniformly picked in
+        [-max_angle, max_angle]
+    """
+    def __init__(self, max_angle: float = 25.) -> None:
+        self.max_angle = max_angle
+
+    def extra_repr(self) -> str:
+        return f"max_angle={self.max_angle}"
+
+    def __call__(self, img: Any, target: Dict[str, np.ndarray]) -> Tuple[Any, Dict[str, np.ndarray]]:
+        angle = random.uniform(-self.max_angle, self.max_angle)
+        rotated_img, rotated_target = F.rotate(img, target, angle)
```

Comment:
Just a thought: would you mind changing back the signature of `rotate` to take not the target but the boxes directly?

This will be much cleaner

Reply:
ok